### PR TITLE
feat(route): 参考消息

### PR DESCRIPTION
**Pull request by yuxinliu-alex**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/cankaoxiaoxi/news/military_news
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/traditional-media.md [hunk 1]
```diff
@@ -521,6 +521,16 @@ Category 列表：
 
 <Route author="boypt"  example="/caixin/yxnews" path="/caixin/yxnews"/>
 
+## 参考消息
+
+### 新闻分类
+
+<Route author="yuxinliu-alex" example="/cancaoxiaoxi/cankaoxiaoxi/news/military_news" path="/cankaoxiaoxi/news/military_news">
```
Comment (TonyRL) [inline docs/traditional-media.md:528]:
```suggestion
<Route author="yuxinliu-alex" example="/cancaoxiaoxi/cankaoxiaoxi/news/military_news" path="/cankaoxiaoxi/news/:category">
```

File: lib/v2/cankaoxiaoxi/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/:category?': ['yuxinliu-alex'],
```
Comment (TonyRL) [inline lib/v2/cankaoxiaoxi/maintainer.js:2]:
```suggestion
    '/news/:category': ['yuxinliu-alex'],
```
This should match `router.js`

File: lib/v2/cankaoxiaoxi/router.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/news/:category?', require('./index'));
```
Comment (TonyRL) [inline lib/v2/cankaoxiaoxi/router.js:2]:
```suggestion
    router.get('/news/:category', require('./index'));
```
`category` looks like a compulsory parameter according to your code

File: lib/v2/cankaoxiaoxi/radar.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'cankaoxiaoxi.com': {
+        _name: '参考消息',
+        '.': [
+            {
+                title: '分类',
+                docs: 'https://docs.rsshub.app/traditional-media.html#can-kao-xiao-xi',
+                source: ['/'],
+                target: '/cankaoxiaoxi/news/:category?',
```
Comment (TonyRL) [inline lib/v2/cankaoxiaoxi/radar.js:9]:
```suggestion
                target: '/cankaoxiaoxi/news/:category',
```

File: lib/v2/cankaoxiaoxi/radar.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'cankaoxiaoxi.com': {
+        _name: '参考消息',
+        '.': [
+            {
+                title: '分类',
```
Comment (TonyRL) [inline lib/v2/cankaoxiaoxi/radar.js:6]:
```suggestion
                title: '新闻分类',
```
This should match H3 heading of the docs

**Review by yuxinliu-alex — COMMENTED** [review]
appreciate for your patient review.

**Comment by yuxinliu-alex** [comment]
Fixed.

**Comment by TonyRL** [comment]
You can use the `category` parameter as subdomain

**Pull request merged**